app/controllers/buyers/users_controller.rb

```diff
@@ -88,7 +88,7 @@ def find_user
   DEFAULT_PARAMS = %i[username email password password_confirmation role].freeze
 
   def user_params
-    @user_params ||= params.require(:user).permit(DEFAULT_PARAMS | user.defined_extra_fields_names)
+    @user_params ||= params.require(:user).permit(*DEFAULT_PARAMS, extra_fields: {})
```

Comment:
To keep validating which fields are allowed, could it be something like?
```suggestion
    @user_params ||= params.require(:user).permit(*DEFAULT_PARAMS, extra_fields: [*user.defined_extra_fields_names])
```

Reply:
Yeah, I had a random blip of thought about doing this, but it was too short for actually adding it :)